locales/en-US/app.ftl

```diff
@@ -186,6 +186,10 @@ newsletterFooterSuccessBody = If you haven't previously confirmed a subscription
 localeWarningTitle = This experiment is only available in English.
 localeWarningSubtitle = You can still enable it if you like.
 
+experimentsListNoneInstalledHeader = Let's get this baby off the ground!
```

Comment:
Would it be possible to add a short localization comment explaining when these strings are displayed (i.e. when no add-ons are installed)?